src/itree.cpp

```diff
@@ -31,13 +38,15 @@ auto print_dir(const fs::path&,
 auto correct_path(const po::variables_map&, rodsEnv&) -> fs::path;
 auto print_usage() -> void;
 auto get_json(const fs::path&, const po::variables_map&, ix::client_connection&, unsigned int depth) -> json::json;
+auto contains_pattern(const pattern_matcher& pm) -> bool;
+
```

Comment:
I know, I know...

I can't help but point it out here, too :)

Reply:
(y)